### PR TITLE
SpaceX React Redux Group Project

### PR DESCRIPTION
**Pull request by yusufobr**
Hello dear,

In this project, my partner and I created the Space Travelers' Hub web-app with the use of React Redux and APIs to fetch and display data. Among the other details of the project are:

- [ ] Use React documentation
- [ ] Use React components
- [ ] Use React Router
- [ ] Connect React and Redux
- [ ] Handle events in a React app
- [ ] Write unit tests with React Testing Library
- [ ] Use styles in a React app
- [ ] Use React hooks
- [ ] Apply React best practices and language style guides in code
- [ ] Use store, actions and reducers in React
- [ ] Perform a code review for a team member

### CODE REVIEW

File: src/Rockets.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```
Comment (t-yanick) [inline src/Rockets.test.js:1]: - [ ] Great job working on this capstone. Kindly take this file _(Rockets.test.js)_ to the `_test_` folder

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @yusufobr and @Bellagirl-maker , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Collaborative work. Commits from both team members
:heavy_check_mark: Effective use of map() to update state
:heavy_check_mark: All pages added and work es expected
:heavy_check_mark: useSelector() used to get state slice corresponding to routes
:heavy_check_mark: Data correctly fetched and Redux store updated accordingly
:heavy_check_mark: Required packages present in the package.json file
:heavy_check_mark: App works as required

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please do well to remove the files generated when you create a new react project that you have not used. I can still see some in your file structure. This will make your project file structure clean professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Rockets.test.js (hunk 1 above)
Comment (yusufobr) [reply src/Rockets.test.js:1]: done

**Review by emmiiorji — APPROVED** [review]
Hi Team, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**